### PR TITLE
First draft for new website announcement news

### LINKED ISSUE #198: Write press release about this new website
anyone volunteering to write a draft? :-) thanks

content:
- short "sightseeing" tour within the new website
- Mention Nicolas Bozon <nicolas.bozon@gmail.com> as the main designer (@nbozon)
- also advertise: mailing lists, wiki, github repos

(needed for #180)

### PR DESCRIPTION
**Pull request by veroandreo**
Please review and edit. We could also add some screenshots afterwards even if ppl will visit it

Fixes #198

### CODE REVIEW

File: content/news/2020_07_29_new_website_and_bday.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## The new GRASS GIS website is out!!!
+
+In ocassion of the 37th birthday and after many years, GRASS GIS website (https://grass.osgeo.org) got again a new and fresh look. It has been redesigned with modern tools to be responsive and also easier to maintain. The content is also more discoverable now and easy to browse.
```
Comment (neteler) [inline content/news/2020_07_29_new_website_and_bday.md:3]:
```suggestion
In occasion of the 37th birthday and after many years, GRASS GIS website (https://grass.osgeo.org) got again a new and fresh look. It has been redesigned with modern tools to be responsive and also easier to maintain. The content is also more discoverable now and easy to browse.
```

File: content/news/2020_07_29_new_website_and_bday.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+### What's cool
+Here a short "sightseeing" tour within the new website:
+ - The "Learn" page offers a curated list of tutorials in different languages and links to videos too. There's also a new "Try online" section with links to Binder and rollApp online applications that allow to test GRASS GIS without installing it.
+ - Our long standing and rich history in the GIS and open source world has now a much attractive layout. Have a look at the timeline of releases and websites, too.
+ - The gallery of screenshots has now a revamped display to show some of GRASS GIS capabilities
+ - Project resources easily reachable: mailing lists, wiki, RSS news feed, the various GitHub and docker repositories.
```
Comment (neteler) [inline content/news/2020_07_29_new_website_and_bday.md:10]:
```suggestion
- The "Learn" page offers a curated list of tutorials in different languages and links to videos too. There's also a new "Try online" section with links to Binder and rollApp online applications that allow to test GRASS GIS without installing it.
- Our long standing and rich history in the GIS and open source world has now a much attractive layout. Have a look at the timeline of releases and websites, too.
- The gallery of screenshots has now a revamped display to show some of GRASS GIS capabilities
- Project resources easily reachable: mailing lists, wiki, RSS news feed, the various GitHub and docker repositories.
```

File: content/news/2020_07_29_new_website_and_bday.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+## The new GRASS GIS website is out!!!
```
Comment (neteler) [inline content/news/2020_07_29_new_website_and_bday.md:1]:
```suggestion
---
title: "New GRASS GIS source code analysis with GitHub Actions"
date: 2020-07-24T12:12:12+01:00
layout: "news"
author: GRASS Development Team
---

## The new GRASS GIS website is out!
```

**Comment by neteler** [comment]
Here a simple screenshot idea, could be added in a lateral way:

image

**Comment by veroandreo** [comment]
I updated the announcement. Here's a screenshot:

image

**Pull request merged**